potato/dagger/Potato/Program.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Diagnostics;
+using System.Text.Json;
+using Dagger.SDK;
+using Client = Dagger.SDK.Query;
+
+internal class Function
+{
+    public required FunctionCall FnCall;
+    public required string ParentName;
+    public required JSON ParentJson;
+    public required string Name;
+    // public required FunctionCallArgValue[] Args;
+
+    public async Task<string> Invoke(Client dag)
+    {
+        var potato = JsonSerializer.Deserialize<Potato.Potato>(ParentJson.Value);
+        Debug.Assert(potato != null, nameof(potato) + " != null");
+        return await potato.Echo("C#");

```

Comment:
I just hit a bug that program got crash when asking input args. So I do temporary hard code argument here to make it works.